packages/amplify-e2e-tests/src/categories/auth.ts

```diff
@@ -404,6 +404,185 @@ export function addAuthWithDefaultSocial(cwd: string, settings: any) {
+      .send('userPoolGroup1')
+      .send('\r')
+      .wait('Do you want to add another User Pool Group')
+      .send('y')
+      .send('\r')
+      .wait('Provide a name for your user pool group')
+      .send('userPoolGroup2')
+      .send('\r')
+      .wait('Do you want to add another User Pool Group')
+      .send('\r')
+      .wait('Sort the user pool groups in order of preference')
+      .send('\r')
+      .wait('Do you want to add an admin queries API?')
+      .send('\r')
+      .wait('Do you want to restrict access to the admin queries API')
+      .send('\r')
+      .wait('Select the group to restrict access with')
+      .send('\r')
+      .wait('Multifactor authentication (MFA) user login options')
+      .send('j')
+      .send('\r')
+      .wait('For user login, select the MFA types')
+      .send('a')
+      .send('\r')
+      .wait('Please specify an SMS authentication message')
+      .send('\r')
+      .wait('Email based user registration/forgot password')
+      .send('\r')
+      .wait('Please specify an email verification subject')
+      .send('\r')
+      .wait('Please specify an email verification message')
+      .send('\r')
+      .wait('Do you want to override the default password policy')
+      .send('y')
+      .send('\r')
+      .wait('Enter the minimum password length for this User Pool')
+      .send('\r')
+      .wait('Select the password character requirements for your userpool')
+      .send('a')
+      .send('\r')
+      .wait('What attributes are required for signing up?')
+      .send('\r')
+      .wait('Specify the app')
+      .send('\r')
+      .wait('Do you want to specify the user attributes this app')
+      .send('y')
+      .send('\r')
+      .wait('Specify read attributes')
+      .send('\r')
+      .wait('Specify write attributes')
+      .send('\r')
+      .wait('Do you want to enable any of the following capabilities?')
+      .send('\r')
+      .wait('Do you want to use an OAuth flow')
+      .send('\r')
+      .wait('What domain name prefix you want us to create for you')
+      .send('\r')
+      .wait('Enter your redirect signin URI')
+      .send('https://signin1/')
+      .send('\r')
+      .wait('Do you want to add another redirect signin URI')
+      .send('n')
+      .send('\r')
+      .wait('Enter your redirect signout URI')
+      .send('https://signout1/')
+      .send('\r')
+      .wait('Do you want to add another redirect signout URI')
+      .send('n')
+      .send('\r')
+      .wait('Select the OAuth flows enabled for this project')
+      .send('\r')
+      .wait('Select the OAuth scopes enabled for this project')
+      .send('\r')
+      .wait('Select the social providers you want to configure for your user pool')
+      .send('a')
+      .send('\r')
+      .wait('Enter your Facebook App ID for your OAuth flow')
+      .send('fbOAUTHid')
+      .send('\r')
+      .wait('Enter your Facebook App Secret for your OAuth flow')
+      .send('fbOAUTHsecret')
+      .send('\r')
```

Comment:
we have an `sendCarriageReturn` function. Lets use that

Reply:
Yup. Done - wasn't aware of the constants.